recipe/bld.bat

```diff
@@ -1,6 +1,6 @@
-set LIB=%LIBRARY_LIB%
-set LIBPATH=%LIBRARY_LIB%;
-set INCLUDE=%LIBRARY_INC%;%PREFIX%\Library\include\freetype2
+set LIB=%LIBRARY_LIB%;%LIB%
+set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
+set INCLUDE=%LIBRARY_INC%;%INCLUDE%
```

Comment:
By moving things in the `freetype` we don't need the special path above. However, I don't think we ever needed as the default channel already moved things if I recall correctly.